### PR TITLE
Ata/censo bootstrap user

### PR DESCRIPTION
**Pull request by ata-peppered**
- Adds logic to create bootstrap key and submit its public key and signature on the wallet-signers call
- Ports SeedRecovery from Prime to Swift; uses 'BigInt' library
- Ports SeedRecoveryTests from Prime to Swift
- Convenience methods to create shares from a ShardingPolicy and rootSeeed
- Convenience methods to create a new ShardingPolicy based on DeviceKey and BootstrapKey
- Adds shards when making the add-wallet-signer call as well as the bootstrap-user call
- New ECPublicKey type where a temp SecKey can be created from the base58 representation of an EC public key and be used for ecies encryption

### CODE REVIEW

File: CensoTests/SeedRecoveryTests.swift [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+                shares: [
+                    newShares[1],
+                    newShares[2],
+                    newShares[3]
+                ]
+            )
+        )
+    }
+
+    func testExample() throws {
```
Comment (bflood-strike) [inline CensoTests/SeedRecoveryTests.swift:224]: nit: do you want to remove this placeholder and the one below

**Review by bflood-strike — APPROVED** [review]
looks awesome. nice job.

File: CensoTests/SeedRecoveryTests.swift (hunk 1 above)
Comment (ata-peppered) [reply CensoTests/SeedRecoveryTests.swift:224]: Yes sir :)

File: Censo/Views/Registration/KeyConfirmationSuccess.swift [hunk 2]
```diff
@@ -24,6 +24,7 @@ struct KeyConfirmationSuccess: View {
     var user: CensoApi.User
     var deviceKey: DeviceKey
     var phrase: [String]
+    var shardingPolicy: ShardingPolicy
```
[the remark this answers is not in the document]
Comment (ata-peppered) [reply Censo/Views/Registration/KeyConfirmationSuccess.swift:27]: Yes I took care of this somewhere else as part of another commit related to the holding screen

**Pull request merged**